light-chat/index.js

```diff
@@ -0,0 +1,166 @@
+import * as utils from "https://unpkg.com/@waku/byte-utils@0.0.2/bundle/index.js";
+import * as wakuCreate from "https://unpkg.com/@waku/create@0.0.4/bundle/index.js";
+import { waitForRemotePeer } from "https://unpkg.com/@waku/core@0.0.6/bundle/lib/wait_for_remote_peer.js";
+import * as wakuMessage from "https://unpkg.com/@waku/core@0.0.6/bundle/lib/waku_message/version_0.js";
+
+const MULTI_ADDR = "/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/443/wss/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm";
+const CONTENT_TOPIC = "/toy-chat/2/huilong/proto";
+const PROTOCOLS = ["filter", "lightpush"];
+
+runApp().catch((err) => {
+    console.error(err);
```

Comment:
Shall we add the error the `Status` section to help? I had an error (usage of crypto in http context only works when using `localhost` and not local lan ip) and it just looked like it tried to connect forever.

Reply:
Good spot, fixed!